dune

```diff
@@ -23,3 +23,5 @@
   (progn
    (bash "diff doc/driver.mld doc/driver.mld.corrected >&2 || true")
    (cat doc/driver-benchmarks.json))))
+
+(vendored_dirs vendor)
```

Comment:
I don't think we want to consider sherlodoc as vendored: we want the tests to be run, the warnings to be displayed, ...